test/EVMHost.cpp

```diff
@@ -58,6 +58,7 @@ evmc::VM& EVMHost::getVM(std::string const& _path)
 				std::cerr << ":" << std::endl << errorMsg;
 			std::cerr << std::endl;
 		}
+		vms[_path]->set_option("validate_eof", "1");
```

Comment:
This is unrelated to the rest of the PR, isn't it? It should not be hidden here. Should be in a separate commit or PR.